### ISSUE TITLE
Resolving relationship relative to part (instead of aasx root)

### ISSUE THREAD

**Issue by seicke** [admin-shell-io/aas-test-engines#14]
I tried to validate an aasx package, generated with [aas4j](https://github.com/eclipse-aas4j/aas4j): [aas-gui.aasx.zip](https://github.com/admin-shell-io/aas-test-engines/files/14901780/aas-gui.aasx.zip)

Got the following output:
<img width="488" alt="1___test_aas-gui___output" src="https://github.com/admin-shell-io/aas-test-engines/assets/2117267/241ce742-a26a-48c3-a240-b93b03572f6f">

Structure of aasx package aas-gui.aasx:
```
aas-gui.aasx
├── _rels
├── aasx
│   ├── _rels
│   │   └── aasx-origin.rels
│   ├── xml
│   │   ├── _rels
│   │   │   └── context.xml.rels
│   │   └── content.xml
│   └── aasx-origin
├── docProps
│   └── core.xml
├── files
│   └── test.pdf
└── [Content_Types].xml
``` 

Content of aasx/_rels/aasx-origin.rels with the named relationship "xml/content.xml":
```
<?xml-version="1.0" encoding="UTF-8" standalone="yes"?>
<Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
    <Relationship Id="afb4e4cfb-287e-4b09-b505-ca8c501aa86b" Target="xml/content xml" Type="http://admin-shell.io/aasx/relationships/aas-spec"/>
</Relationships>
``` 

It seems that the validation tool only resolve relationships relative to the AASX root (and not relative to parts, in this case "aasx")

This issue is related/comparable to https://github.com/admin-shell-io/aasx-package-explorer/issues/654

**Comment by otto-ifak** [comment]
Hello @seicke and thanks for reporting this.
We fixed your issue with 551f7e4dc55800a66c1ed7a9fb2cb0166362ef4d

However, you aasx file is still rejected. Do you expect it to be compliant to v3.0 of the meta model?

**Comment by seicke** [comment]
Hello @otto-ifak,
thanks for your quick response and for fixing!

Handling of relative paths is working now, thanks again!

PS: I did not expect that the AASX is compliant to v3.0, I expected a rejection (but not caused by relative paths 😃)
